applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -372,22 +344,25 @@ int ext_sensors_accelerometer_trigger_callback_set(bool enable)
 	int err;
 	struct sensor_trigger trig = {
 		.chan = SENSOR_CHAN_ACCEL_XYZ,
-		.type = SENSOR_TRIG_THRESHOLD
+		.type = SENSOR_TRIG_MOTION
 	};
 	struct ext_sensor_evt evt = {0};
 
 	sensor_trigger_handler_t handler = enable ? accelerometer_trigger_handler : NULL;
 
 	err = sensor_trigger_set(accel_sensor.dev, &trig, handler);
 	if (err) {
-		LOG_ERR("Could not set trigger for device %s, error: %d",
-			accel_sensor.dev->name, err);
-		evt.type = EXT_SENSOR_EVT_ACCELEROMETER_ERROR;
-		evt_handler(&evt);
-		return err;
+		goto error;
 	}
-
-	initial_trigger = false;
-
-	return 0;
+	trig.type = SENSOR_TRIG_STATIONARY;
+	err = sensor_trigger_set(accel_sensor.dev, &trig, handler);
+	if (!err) {
```

Comment:
Why not just `goto err` on error and have the `return 0` after the if statement. Then we also get the error message printed.